bruno/Sample Data/Resouces/Schools/02 - Schools.bru

```diff
@@ -0,0 +1,110 @@
+meta {
+  name: 02 - Schools
+  type: http
+  seq: 2
+}
+
+post {
+  url: {{resourceBaseUrl}}/ed-fi/schools
+  body: json
+  auth: bearer
+}
+
+auth:bearer {
+  token: {{edFiCertToken}}
+}
+
+body:json {
+  {
+    "id": null,
+    "schoolId": "{{tempSchoolId}}",
+    "nameOfInstitution": "Saint Joseph High School",
+    "shortNameOfInstitution": "SJHS",
+    "webSite": "https://www.google.com",
+    "operationalStatusDescriptor": "uri://ed-fi.org/OperationalStatusDescriptor#Active",
+    "schoolTypeDescriptor": "uri://ed-fi.org/SchoolTypeDescriptor#Regular",
+    "administrativeFundingControlDescriptor": "uri://ed-fi.org/AdministrativeFundingControlDescriptor#Private School",
+    "charterStatusDescriptor": "uri://ed-fi.org/CharterStatusDescriptor#Not a Charter School",
+    "titleIPartASchoolDesignationDescriptor": "uri://ed-fi.org/TitleIPartASchoolDesignationDescriptor#Not A Title I School",
+    "localEducationAgencyReference": {
+      "localEducationAgencyId": "{{tempLEAId}}"
+    },
+    "educationOrganizationCategories": [{
+      "educationOrganizationCategoryDescriptor": "uri://ed-fi.org/EducationOrganizationCategoryDescriptor#School"
+    }],
+    "addresses": [{
+      "addressTypeDescriptor": "uri://ed-fi.org/AddressTypeDescriptor#Physical",
+      "city": "Saint Joseph",
+      "postalCode": "99999",
+      "stateAbbreviationDescriptor": "uri://ed-fi.org/StateAbbreviationDescriptor#TX",
+      "streetNumberName": "456 Elm Street",
+      "nameOfCounty": "Williston"
+    }, {
+      "addressTypeDescriptor": "uri://ed-fi.org/AddressTypeDescriptor#Mailing",
+      "city": "Saint Joseph",
+      "postalCode": "99999-9999",
+      "stateAbbreviationDescriptor": "uri://ed-fi.org/StateAbbreviationDescriptor#TX",
+      "streetNumberName": "P.O. Box 2035",
+      "nameOfCounty": "Williston"
+    }],
+    "identificationCodes": [{
+      "educationOrganizationIdentificationSystemDescriptor": "uri://ed-fi.org/EducationOrganizationIdentificationSystemDescriptor#School",
+      "identificationCode": "{{tempSchoolIdCode}}"
+    }],
+    "indicators": [{
+      "indicatorDescriptor": "uri://gbisd.edu/IndicatorDescriptor#Retention Rate",
+      "indicatorGroupDescriptor": "uri://gbisd.edu/IndicatorGroupDescriptor#Staff Indicator",
+      "indicatorLevelDescriptor": "uri://gbisd.edu/IndicatorLevelDescriptor#High Retention",
```

Comment:
You had no way of knowing this, but some systems won't have the `gbisd.edu` namespace. This represents a fake school district, "Grand Bend Independent School District," and these values are from a sample extension.

I recommend changing to some descriptors in the `ed-fi.org` namespace, just to be on the safe side.